_saas-integrations/pardot/pardot 15-10-2015.md

```diff
@@ -1,10 +1,11 @@
 ---
 title: Pardot
-permalink: /integrations/saas/pardot
+permalink: /integrations/saas/pardot/v15-10-2015
 keywords: pardot, pardot data, etl pardot, pardot etl, pardot schema
 summary: "Connection instructions and schema details for Stitch's Pardot integration."
 layout: singer
 old-schema-template: true
+input: false
```

Comment:
Old versions will also need this flag added (or more likely, uncommented). Otherwise multiple versions of the integration will display in the sidenav.